<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titleMenu</class>
 <widget class="QWidget" name="titleMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="menuButton" name="ButtonInfo" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="menuButton" name="ButtonFile" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="menuButton" name="ButtonWord" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>menuButton</class>
   <extends>QWidget</extends>
   <header>menuButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
